packages/fast-colors/src/color-converters.ts

```diff
@@ -63,6 +63,28 @@ export function rgbToLinearLuminance(rgb: ColorRGBA64): number {
     return rgb.r * 0.2126 + rgb.g * 0.7152 + rgb.b * 0.0722;
 }
 
+/**
+ * Calculate an overlay color that uses rgba (rgb + alpha) that matches the appareance of a given solid color when placed on the same background
+ * @param rgbMatch The solid color the overlay should match in appearance when placed over the rgbBackground
+ * @param rgbBackground The background on which the overlay rests
+ * @param rgbOverlay The rgb color of the overlay. Typically this is either pure white or pure black. This color will be used in the returned output
+ * @returns The rgba (rgb + alpha) color of the overlay
+ */
+export function calculateOverlayColor(

```

Comment:
Can you add a couple test cases to this so that we can make a few known assertions?